dist/SoftLedgerAPI.js

```diff
@@ -333,6 +333,9 @@ class SoftLedgerAPI {
 	getStatusOfCostBasisCalculation() {
 		return this.instanceV2.get('/status/inventory');
 	}
+    getSalesOrderCreatedAuditLogBySOId(params) {
```

Comment:
this function is getting any audit log data matching the specific params right? not specifically related to sales order created audit logs? 

it may be better to rename this function to be more generic -- something like getAuditLogsForParams